### PR TITLE
Adding assignment and ownership of work items to CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by markito**
Signed-off-by: William Markito Oliveira <markito@redhat.com>

### CODE REVIEW

**Comment by markito** [comment]
@duglin - Had to close the previous PR and open a new one to make sure sign-off was working properly.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -26,6 +26,12 @@ the guidelines we follow for how documents are formatted.
 
 Each PR must be signed per the following section.
 
+### Assigning and Owning work
+
+If you want to own and work on an issue, add a comment or “#dibs” it asking 
+about ownership. Contributors will then add the "Assigned" label with 
+“Assigned to: person@xxx”.
```
Comment (duglin) [inline CONTRIBUTING.md]: A maintainer will then add the `Assigned` label and modify the first comment
in the issue to include `Assigned to: @person`.

**Comment by duglin** [comment]
@markito no worries, but next time I think you can add your signature by just doing:
```
git commit -s --amend --no-edit
git put origin BRANCH-NAME -f
```

**Comment by duglin** [comment]
LGTM

**Comment by ac360** [comment]
Nice, @markito.  LGTM.

**Comment by duglin** [comment]
Approved on 2/8 call

**Pull request merged**